libraries/AC_Avoidance/AC_Avoid.cpp

```diff
@@ -79,6 +79,22 @@ const AP_Param::GroupInfo AC_Avoid::var_info[] = {
     // @Range: 0 2
     // @User: Standard
     AP_GROUPINFO("BACKUP_SPD", 6, AC_Avoid, _backup_speed_max, 0.5f),
+    
+    // @Param: ALT_MIN
+    // @DisplayName: Avoidance minimum altitude
+    // @Description: Minimum altitude above which avoidance will start working. This requires a valid downward facing rangefinder reading to work. Set zero to disable
```

Comment:
is all avoidance disabled below this altitude or just proximity?  if it's just proximity based avoidance then we should clarify that in this description.